client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/site-preview/hooks/test/use-site-preview-mshot-image-handler.test.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @jest-environment jsdom
+ */
+import { renderHook, act } from '@testing-library/react';
+import { useSitePreviewMShotImageHandler } from '../use-site-preview-mshot-image-handler';
+
+// Mocking fetch globally
+global.fetch = jest.fn( () =>
```

Comment:
Maybe it would be better to use `nock` here instead?

Reply:
I tried a lot but it wasn't working. Then got the "TIL" that native fetch doesn't work with nock! Updated here using HTTP request https://github.com/Automattic/wp-calypso/pull/92355/commits/a4800bcf8f711518f2a590aec036f67e621d357f

Reply:
Oh! TIL!
Also, after seeing your change, should we update it to use `wp.req`, similar to the other uses in Calypso?

Reply:
We are calling s0.wp.com here, not sure wp.req can be used for these endpoints 🤔

Reply:
Ah, sorry! I missed this, and I didn't see that other places in Calypso are calling it this way. So it's fine! =)
Thank you for checking and letting me know!